uma/uma.py

```diff
@@ -73,6 +74,36 @@ def fetch_public_key_for_vasp(
     return public_key
 
 
+def create_pubkey_response(
+    signing_cert_chain: str,
+    encryption_cert_chain: str,
+    expiration_timestamp: Optional[datetime] = None,
+) -> PubkeyResponse:
+    """
+    Creates a pubkey response.
+
+    Args:
+        signing_cert_chain: the chain of signing certificates in PEM format.
+            The order of the certificates should be from the leaf to the root.
```

Comment:
leaf to node order seems more conventional (https://www.rfc-editor.org/rfc/rfc5246#section-7.4.2)